packages/teams-js/src/public/interfaces.ts

```diff
@@ -169,6 +169,54 @@ export enum FileOpenPreference {
   Web = 'web',
 }
 
+export enum ActionObjectType {
+  M365Content = 'm365content',
+}
+
+export interface BaseActionObject<T extends ActionObjectType> {
```

Comment:
This also needs documentation comments. If it's appropriate to keep it hidden from public docs, please add the @hidden tag.